src/main.ts

```diff
@@ -6,45 +6,59 @@ import {
   generateRowKey,
 } from "./core/structure/key/keyTypes.ts";
 import { CellInfo } from "./core/structure/sheet.types.ts";
-
+import { GenerateRowLabel } from "../utils/helpers.ts";
 export default class LightSheet {
   ui: UI;
   options: LightSheetOptions;
   sheet: Sheet;
   onCellChange?;
 
+  #defaultRowCount: number = 4;
+  #defaultColCount: number = 4;
+
```

Comment:
Shouldn't these be in `LightsheetOptions`? Maybe parameters for table width/height with default values of 4.

Reply:
Yes it makes sense. Thanks.